src/etl/disease_etl.py

```diff
@@ -70,6 +71,7 @@ class DiseaseETL(ETL):
 
             MERGE (dga:Association:DiseaseEntityJoin {primaryKey:row.diseaseUniqueKey})
                 SET dga.dataProvider = row.dataProvider
+                SET dga.dateAssigned = row.dateAssigned
```

Comment:
These SET's should be combined into one SET with comma's separating the variables.